src/TMap.cpp

```diff
@@ -1532,7 +1532,7 @@ bool TMap::restore( QString location )
         }
     }
 
-    if( ! canRestore || entries.size() == 0 ) {
+    if( (! canRestore || entries.size() == 0) && downloadIfNotFound ) {
```

Comment:
For any MUD that supports MXP map format - which is IRE MUDs only... I wish more of them supported it!